pkg/util/initsystem/initsystem.go

```diff
@@ -166,8 +228,12 @@ func (sysd WindowsInitSystem) ServiceIsActive(service string) bool {
 // if we cannot detect a supported init system for pre-flight checks.
 // This indicates we will skip init system checks, not an error.
 func GetInitSystem() (InitSystem, error) {
+	_, err := exec.LookPath("openrc")
```

Comment:
Does the order matter? i mean ... can we leave the check for `systemctl` as the first check ( to keep existing behavior when both are accidentally present in a system? )

Reply:
@dims order does not matter. I just saw that `wininit` comes after `systemctl`. I thought it was intentionally alphabetically orders.
I will change this as you asked.